[PiT 24.7]starter is not working with 24.7.0.alpha12

use [this branch](https://github.com/vaadin/skeleton-starter-flow-cdi/tree/test-vaadin-24.7.0.alpha12) to reproduce the issue

run `mvn wildfly:run`
then navigate to 'localhost:8080'

seems a jackson dependency is missing from the war
```
Caused by: java.lang.NoSuchMethodError: 'com.fasterxml.jackson.core.util.Separators com.fasterxml.jackson.core.util.Separators.withObjectFieldValueSpacing(com.fasterxml.jackson.core.util.Separators$Spacing)'
        at deployment.project-base-flow-cdi-1.0-SNAPSHOT.war//com.vaadin.flow.internal.JacksonUtils.toFileJson(JacksonUtils.java:510)
        at deployment.project-base-flow-cdi-1.0-SNAPSHOT.war//com.vaadin.flow.server.frontend.NodeUpdater.writePackageFile(NodeUpdater.java:524)
        at deployment.project-base-flow-cdi-1.0-SNAPSHOT.war//com.vaadin.flow.server.frontend.NodeUpdater.writePackageFile(NodeUpdater.java:518)
        at deployment.project-base-flow-cdi-1.0-SNAPSHOT.war//com.vaadin.flow.server.frontend.TaskGeneratePackageJson.execute(TaskGeneratePackageJson.java:58)

```

![Image](https://github.com/user-attachments/assets/477dc555-d787-42a7-8fc4-bacfcf9591f8)

--- a/pom.xml
+++ b/pom.xml
@@ -100,9 +100,9 @@
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
                 <artifactId>wildfly-maven-plugin</artifactId>
-                <version>4.0.0.Final</version>
+                <version>5.1.2.Final</version>
                 <configuration>
-                    <version>27.0.1.Final</version>
+                    <version>35.0.1.Final</version>
                 </configuration>
             </plugin>
 

